Client/Frontend/Fakespot/Views/FakespotReviewQualityCardView.swift

```diff
@@ -174,9 +174,9 @@ final class FakespotReviewQualityCardView: UIView, Notifiable, ThemeApplicable {
 
     private lazy var highlightsLabel: UILabel = .build { label in
         let text = String.Shopping.ReviewQualityCardHighlightsLabel
```

Comment:
According the string inventory this string needs to be adjusted according the retailer that is currently visited (Amazon, Walmart, Best Buy).

Reply:
I wanted to use what was already done for the websites order, but maybe I should move the PartnerWebsite enum to the FakespotViewModel instead?

Reply:
Let's move `PartnerWebsite` to its own file in the Fakespot/Client folder.